src/containers/Sidebar/index.js

```diff
@@ -8,15 +8,17 @@ import SidebarBottomItem from './SidebarBottomItem';
 import {COLLAPSE_SIDEBAR} from "../../actions";
 
 
-const Sidebar = (props) => {
+export default function Sidebar (props) {
```

Comment:
@Aringoaway Why did you make a function out of it? Just a question, I dont say its wrong